artifactory/services/upload.go

```diff
@@ -448,6 +469,7 @@ type UploadParams struct {
 	Flat              bool
 	Retries           int
 	MinChecksumDeploy int64
+	VcsData           *clientutils.VcsDetails
```

Comment:
There's probably no need to store VcsData as an UploadParam.

Reply:
Removed, add flag instead.